### PR TITLE
Anaconda updates f33 support

### PR DESCRIPTION
**Pull request by jkonecny12**
New Fedora was branched so add support here.

### CODE REVIEW

File: anaconda_updates/scripts/show_version.sh [hunk 1]
```diff
@@ -62,6 +62,10 @@ do
          PACKAGES_URL="fedora32/Packages/a"
          shift
          ;;
+        -f33)
+         PACKAGES_URL="fedora32/Packages/a"
```
Comment (VladimirSlavik) [inline anaconda_updates/scripts/show_version.sh]: 33 ?

Comment (jkonecny12) [reply anaconda_updates/scripts/show_version.sh]: fixed, thanks!

**Comment by jkonecny12** [comment]
**UPDATED:**
* Fix mistake pointed by @VladimirSlavik

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

**Pull request merged**